swagger-ui/swagger-ui.js

```diff
@@ -42,6 +42,18 @@ this["Handlebars"]["templates"]["content_type"] = Handlebars.template({"1":funct
 },"useData":true});
 'use strict';
 
+var replaceResponse = function(a, b, element) {
```

Comment:
consider putting this function in its own Javascript document and then modify this file only to call it